app/src/main/java/edu/wpi/first/shuffleboard/app/components/TilePane.java

```diff
@@ -409,6 +409,9 @@ public boolean isOverlapping(int col, int row, int tileWidth, int tileHeight, Pr
     return false;
   }
 
+  /**
+   * Gets the layout of a tile in this tile pane.
```

Comment:
Gets or creates??

Reply:
It computes the layout and stores it in a data object? It only "creates" a layout in the very narrow sense that a new data object is created to contain it